src/equations/equations.jl

```diff
@@ -18,8 +18,9 @@ end
 ####################################################################################################
 # Include files with actual implementations for different systems of equations. 
 
-include("shallow_water_wet_dry_1d.jl")
-include("shallow_water_wet_dry_2d.jl")
+include("shallow_water_quasi_1d.jl")
+include("shallow_water_1d.jl")
+include("shallow_water_2d.jl")
```

Comment:
```suggestion
include("shallow_water_quasi_1d.jl")

include("shallow_water_1d.jl")
include("shallow_water_2d.jl")
```
Just for consistency separating the different physics.